  0%|                                                                                                                                                                             | 0/500 [00:00<?, ?it/s]  0%|                                                                                                                                                                             | 0/500 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "ORIEL/Metro-Master/metro_training.py", line 438, in <module>
  File "/sfs/lustre/bahamut/scratch/at5cf/ORIEL/Metro-Master/src/utils/model_training_utlis_wtensorboard.py", line 193, in train_model
    for batch_idx, batch in enumerate(train_dataloader):
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 345, in __next__
    data = self._next_data()
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 856, in _next_data
    return self._process_data(data)
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 881, in _process_data
    data.reraise()
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/_utils.py", line 395, in reraise
    raise self.exc_type(msg)
FileNotFoundError: Caught FileNotFoundError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/utils/data/_utils/worker.py", line 178, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/sfs/lustre/bahamut/scratch/at5cf/ORIEL/Metro-Master/src/dataloaders/uva_dar_dataset.py", line 181, in __getitem__
    seq, seq_len = self.get_video_data(idx, modality)
  File "/sfs/lustre/bahamut/scratch/at5cf/ORIEL/Metro-Master/src/dataloaders/uva_dar_dataset.py", line 149, in get_video_data
    seq = torch.load(data_filepath, map_location='cpu').detach()
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/serialization.py", line 584, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/serialization.py", line 234, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/at5cf/.conda/envs/oriel/lib/python3.6/site-packages/torch/serialization.py", line 215, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/scratch/at5cf/ann/fe_embed/working_with_phone/2019_0607_163008_006B.MP4_188_190_.pt'

